src/v2/Apps/Auctions/Components/WorksByArtistsYouFollowRail/WorksByArtistsYouFollowRail.tsx

```diff
@@ -23,7 +23,20 @@ const WorksByArtistsYouFollowRail: React.FC<WorksByArtistsYouFollowRailProps> =
   const nodes = extractNodes(viewer.saleArtworksConnection)
 
   if (nodes.length === 0) {
-    return null
+    return (
+      <Box>
+        <Text
+          as="h3"
+          color="black60"
+          mb={12}
+          mt={6}
+          textAlign="center"
+          variant="mediumText"
+        >
+          No Works To Show
+        </Text>
+      </Box>
```

Comment:
Since this bit is duplicated a few times we could probably extract it into a `CuritorialRailZeroState` component and share it across all three spots

Reply:
That seems to me like it would be a pretty small component, especially as the only text in it would be a 5 lines worth of code....

Is that worth it?

Reply:
Small components are fine; thinking: why make three changes to three places when you can make one change to one place and all three locations update as if by magic? 

More generally tho, I stick pretty religiously to the [Rule of Three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) axiom. If there were just two spots i'd probably say meh, but since there are three it pings this rule and led me to this comment.

Reply:
Added `CuratorialRailsZeroState`